wordle-web/layouts/default.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-app>
+    <v-app-bar color="primary" :elevation="2">
+      <v-app-bar-title @click="$router.push('/')" style="cursor: pointer">
+        <v-icon> mdi-flower-poppy </v-icon>
+        Aesthetic Wordle</v-app-bar-title
+      >
+
+      <v-btn icon="mdi-help-circle" @click="showHelpDialog = true" />
+      <v-btn icon="mdi-cog" @click="showSettingsDialog = true" />
+      <v-app-bar-nav-icon
+        variant="text"
+        @click="drawer = !drawer"
+      ></v-app-bar-nav-icon>
+    </v-app-bar>
+    <v-navigation-drawer
+      v-model="drawer"
+      location="right"
+      color="secondary"
+      temporary
+    >
+      <v-list v-for="item in ['About']" :key="item">
```

Comment:
I like this implementation. It feels more reusable than my hardcoded solution

Reply:
Thanks. I honestly thought it was weird to do this since we only have one item in the menu right now. I also feel like there may be a better way where you can pull each page programmatically somehow instead of having to keep adding to it new routes in. If that makes sense.